### ISSUE TITLE
[New App Shortcut]: Weyd  

### ISSUE THREAD

**Issue by CodyJon** [PRProd/HA-Firemote#192]
### App Name

Weyd

### Device Platform

NVIDIA Shield

### App Store Link or URL to .apk file if sideloaded

https://weyd.app/d

### Application Info

```
mSurface=Surface(name=app.weyd.player/app.weyd.player.ui.LinkResolveActivity)/@0xc81abc3
        mSurface=Surface(name=com.android.systemui.ImageWallpaper)/@0x4b3ffe5
    mInputMethodTarget in display# 0 Window{dab7b6a u0 app.weyd.player/app.weyd.player.ui.LinkResolveActivity}
```

### Optional Comments

Wanted to just say a huge thank you for the work on this project! :)

**Comment by PRProd** [comment]
Greetings, giver of coffee!  :smile:  (thanks again, BTW)

I've just released version [v2.3.16](https://github.com/PRProd/HA-Firemote/releases/tag/v2.3.16) containing your app launch request!

Enjoy!

**Comment by CodyJon** [comment]
> Greetings, giver of coffee!  :smile:  (thanks again, BTW)
> 
> 
> 
> I've just released version [v2.3.16](https://github.com/PRProd/HA-Firemote/releases/tag/v2.3.16) containing your app launch request!
> 
> 
> 
> Enjoy!

Amazing!! Thank you again, looks great :)
